### PR TITLE
Update LostOutlaws manifest.json

### PR DESCRIPTION
**Pull request by zOIDAdaJulian**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/lostoutlaws/manifest.json [hunk 1]
```diff
@@ -1,13 +1,23 @@
 {
   "server_name": "lostoutlaws",
   "nice_name": "LostOutlaws",
-  "direct_ip": "minecraft.lostoutlaws.de",
+  "direct_ip": "lostoutlaws.de",
   "server_wildcards": [
     "%.lostoutlaws.de"
   ],
   "social": {
+	"web_support": "https://discord.lostoutlaws.de",
```
Comment (JakobBruening) [inline minecraft_servers/lostoutlaws/manifest.json:9]: Web support means something like a faq or a support web page, not a discord link. Please remove the line.

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/lostoutlaws/manifest.json (hunk 1 above)
Comment (zOIDAdaJulian) [reply minecraft_servers/lostoutlaws/manifest.json:9]: Ah ok then I misunderstood that.
I'll change it as fast as I'm home again! ^^

**Comment by zOIDAdaJulian** [comment]
Ok, the manifest should be ok again :D

**Review by JakobBruening — APPROVED** [review]
Nice! Thank you!

**Pull request merged**